core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -1599,4 +1629,28 @@ object KafkaZkClient {
       time, metricGroup, metricType)
     new KafkaZkClient(zooKeeperClient, isSecure, time)
   }
+
+
+  def controllerZkVersionCheck(version: Int): Option[ZkVersionCheck] = {
+    if (version < 0)
+      None
+    else
+      Some(ZkVersionCheck(ControllerEpochZNode.path, version))
+  }
+
+  def maybeThrowControllerMoveException(response: AsyncResponse): Unit = {
+    response.zkVersionCheckResult match {
+      case Some(zkVersionCheckResult) =>
+        val zkVersionCheck = zkVersionCheckResult.zkVersionCheck
+        if (zkVersionCheck.checkPath.equals(ControllerEpochZNode.path))
```

Comment:
It seems that we need extra indentation for the body of the `if` statement.

Reply:
Fixed.